app/konnect/dev-portal/customization/self-hosted-portal.md

```diff
@@ -0,0 +1,49 @@
+* **Self-hosted, open source Dev Portal**: Provides all the same features as the cloud-hosted portal, but also gives you control over your hosting service. 
+
+You can use the open source Dev Portal to display your APIs to developers on a self-hosted website. This page explains the benefits of using a self-hosted Dev Portal, a high-level overview of how the self-hosted portal works, and how to enable it.
+
+## Self-hosted Dev Portal benefits
+
+There are several benefits to keep in mind when deciding whether to use a {{site.konnect_short_name}}-hosted or self-hosted Dev Portal. The self-hosted portal provides the following benefits: 
+
+* **Fully customizable:** Use the [example frontend Dev Portal application](https://github.com/Kong/konnect-portal) as a starting point and then customize Dev Portal for your needs using the [Portal API](https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356). You can also integrate the API specs with workflows tailored to your organization's own processes.
+* **Hosting service choice:** When you self-host, you also get to choose which hosting service you use to deploy your Dev Portal. 
+* **Range of customization options:** With the self-hosted Dev Portal, you determine how much you want to customize. You can choose to use the example application right out of the box, or you can use the [Portal API](https://developer.konghq.com/spec/2aad2bcb-8d82-43b3-abdd-1d5e6e84dbd6/b4539157-4ced-4df5-affa-7d790baee356) and [Portal SDK](https://www.npmjs.com/package/@kong/sdk-portal-js) for more fine-grained control.
+
+## How the self-hosted Dev Portal works 
+
+The following diagram explains how the self-hosted Dev Portal works by using the example frontend application:
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
+
+> Figure 1: Diagram that shows how the example frontend application communicates with the developer's browser and {{site.konnect_short_name}}. The self-hosted Dev Portal is hosted in your infrastructure and the application is served to the developer browser. The developer browser communicates with the Portal API for API requests and {{site.konnect_short_name}} supplies the corresponding content. 
+
+The Dev Portal API is always hosted by {{site.konnect_short_name}}, no matter if you use the self-hosted or {{site.konnect_short_name}}-hosted Dev Portal. The Dev Portal API maps to the Portal Domain (and optional Custom Hosted Domain, which is a CNAME). Depending on which domain settings you configure and if you are self-hosting or not, the Dev Portal UI domain will differ.
+
+![DIAGRAM HERE](/assets/images/docs/konnect/diagram.png)
+
+> Figure 2: Diagram that shows that self-hosted and {{site.konnect_short_name}}-hosted Dev Portals use different UI domains, but both use the same {{site.konnect_short_name}}-hosted Dev Portal API.
```

Comment:
@Guaris If you have a chance, could you review this? I added lines 28-32 to accommodate a diagram Ross wanted to add. I hope that it makes sense even without the diagram added. Hopefully marketing wraps up those diagrams soon.

Reply:
Oh, and if that looks good (I also read through the whole thing again), could you approve this PR so I can hand it over to Ross?